build/NuSpecs/MUXControls.nuspec

```
@@ -28,6 +41,10 @@
     <file target="lib\uap10.0\Microsoft.UI.Xaml\Themes" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml\Generic.xaml"/>
     <file target="lib\uap10.0\Design" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.Design\Microsoft.UI.Xaml.design.dll"/>
 
+    <file target="lib\net8.0-windows10.0.22621.0\Microsoft.UI.Xaml.Projection.dll" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml.Projection\Microsoft.UI.Xaml.Projection.dll" />
+    <file target="lib\net8.0-windows10.0.22621.0\Microsoft.UI.Xaml\Themes" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.UI.Xaml\Generic.xaml"/>
+    <file target="lib\net8.0-windows10.0.22621.0" src="Intellisense\Microsoft.UI.Xaml.xml"/>
```

We need to match the name of this .xml to the projection .dll (same for the other .nuspec files):
```suggestion
    <file target="lib\net8.0-windows10.0.22621.0\Microsoft.UI.Xaml.Projection.xml" src="Intellisense\Microsoft.UI.Xaml.xml"/>
```